File: Clase1/ejemplos/test1.py
def calcularCuadrado(x):
	return x*x

numeros = range(0,10000000)
salida = []

#for n in numeros:
#	salida.append(calcularCuadrado(n))
salida = map(calcularCuadrado,numeros)


#print(numeros)
#for n in salida:
#	print(n)

File: Clase1/ejemplos/test1.c
#include <stdio.h>

#define LEN	10000000

static long numeros[LEN];
static long salida[LEN];

static long calcularCuadrado(long x)
{
	return x*x;
}

int main(void)
{
	long i;
	for(i=0; i<LEN; i++)
		numeros[i]=i;	

	for(i=0; i<LEN; i++)
		salida[i]=calcularCuadrado(numeros[i]);	
	

	return 0;
}

